### ISSUE TITLE
Can't build Safari extension

### ISSUE THREAD

**Issue by quantumgolem** [yimingliu/reverse-image-search-safari#15]
So I'm trying to compile this from source using Xcode 12 beta. When building, I can get the app to build and run fine. However, I'm not sure how to get the Safari extension working, as it doesn't "open" automatically when I build the project. 

I tried looking at the destination of the built files. There is the `.app`, which I can run and it opens the normal app view. There is also a different file with extension `.appex` which appears to be the Safari extension. However, when I double click on it, it tried to open with QuickLookSimulator. It does not seem as though I can open it with Safari. 

How do I build and install the Safari extension?

**Comment by yimingliu** [comment]
To start with, I am currently not supporting XCode 12 beta or Big Sur beta (as I am not running them myself yet).  The following applies to Catalina and XCode 11 - YMMV.

The extension should be automatically installed in Safari (in a deactivated state) when you run the containing .app the first time.  Then you have to turn it on from the Extensions part in Safari Preferences.

Debugging questions:
- Does the extension appear in Safari menu -> Preferences -> Extensions, after you run the containing app?
- Do you have an Apple Developer certificate for code signing?
- Did the [prebuilt binary](https://github.com/yimingliu/reverse-image-search-safari/releases) work?

**Comment by quantumgolem** [comment]
Hey, thanks a lot for your reply! I looked at your debugging steps and finally found out that it must be because I wasn't signing it properly (I was running it "locally" because I don't have a paid Apple developer account). I investigated this a bit further and it seems like when I enable unsigned extensions in Safari I can see it and can enable it from my preferences for the duration of the session. So now everything works! Thanks again for your help :)
